pulpcore/cli/rpm/publication.py

```diff
@@ -37,18 +45,21 @@ def publication(ctx: click.Context, pulp_ctx: PulpContext, publication_type: str
 
 lookup_options = [href_option]
 create_options = [
+    repository_option,
+    click.option(
+        "--version", type=int, help=_("a repository version number, leave blank for latest")
+    ),
+]
+filter_options = publication_filter_options + [
     resource_option(
         "--repository",
         default_plugin="rpm",
         default_type="rpm",
         context_table={"rpm:rpm": PulpRpmRepositoryContext},
-    ),
```

Comment:
Should this be the same `repository_option` now?

Reply:
Not sure where this comment is pointing to.

Reply:
The git diff is a bit wacky, but the difference between the two is that the filter option requires pulpcore 3.20, while the create option does not.

Reply:
Ah, thank you.